script/Ciel/Day 11/11_01_CIEL11_2_1.txt

```diff
@@ -339,7 +357,7 @@ Let's presume we then also have Being B, who has had their lifeforce stolen by B
 [sha:019c5c9848595318ed7459dba11617384a3498f0]{
 -- Page 116, Offset 29393.
 -- 吸血鬼と人間の関係のようだが、本質は違う。
-The essence is different, but it's almost like the relationship between humans and vampires.
+It differs in the details, but it's almost like the relationship between humans and vampires.
```

Comment:
Not really the details, so much as the nature of the relationship is fundamentally different, even if they seem alike on the surface?

>The fundamental nature of it is different, but it does resemble the...